rep-2000.rst

```diff
@@ -892,43 +892,43 @@ Minimum language requirements:
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Ignition    |    Fortress*   |       N/A      |   N/A    |    Fortress*   |   Fortress*   |     Fortress*    |         N/A         |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Ogre        |                                     1.10*                                                      |         N/A         |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| OpenCV      |     4.5.4      |     4.5.4      |  3.4.6   |     4.2.0      |     4.2.0     |      4.5.1       |  4.1.0 / 3.2.0***   |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| OpenSSL     |     1.1.1l     |     1.1.1l     |  1.1.1g  |     1.1.1d     |     1.1.1f    |      1.1.1i      | 1.1.1d / 1.1.1b***  |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Python      |     3.10.0     |     3.8.3      |  3.6.8   |     3.8.0      |     3.8.2     |      3.9.1       |  3.8.2 / 3.7.5***   |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Qt          |     5.15.2     |    5.12.10     |  5.12.5  |     5.12.5     |     5.12.3    |      5.15.2      | 5.14.1 / 5.12.5***  |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+|                              |                                                    **Linux only**                                   |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| PCL         |     1.12.0     |      N/A       |  1.12.0  |     1.10.0     |      N/A      |      1.11.1      |       1.10.0        |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+|                                         **RMW DDS Middleware Providers**                                                           |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Cyclone DDS |                                          0.8.x (Réplique)                                                            |
++-------------+----------------+----------------+----------+----------------+---------------+------------------+---------------------+
+| Fast-DDS    |              2.4.x                         |     2.3.x      |   2.4.x                                                |
```

Comment:
Ah sorry, I missed this last time.  We are going to support the same version of Fast-DDS across the board here, so you can just make this entire row `2.4.x`.

Reply:
Fixed in https://github.com/ros-infrastructure/rep/pull/342/commits/d0a6e6386c66fc31c94f224384e9c67d4afaa65e.